splice_machine/src/main/java/com/splicemachine/derby/impl/SpliceService.java

```diff
@@ -61,8 +62,10 @@ public Properties getServiceProperties(String serviceName, Properties defaultPro
 			Set<String> properties = propertyManager.listProperties();
 //			List<String> children = ZkUtils.getChildren(zkSpliceDerbyPropertyPath, false);
 			for (String property: properties) {
-				String value = propertyManager.getProperty(property);
-				service.setProperty(property, value);
+				if (!property.equals(Property.SELECTIVITY_ESTIMATION_INCLUDING_SKEWED)) {
```

Comment:
please add a comment why we need special handling of SELECTIVITY_ESTIMATION_INCLUDING_SKEWED here. if this is because SELECTIVITY_ESTIMATION_INCLUDING_SKEWED is now not stored in propertyManager anymore, then i think we should have sth like `PropertyUtil.deletedPropertiesList`